.storybook/FormField.js

```diff
@@ -179,6 +188,72 @@ storiesOf('InputField', module)
       </Flex>
     ))
   )
+  .add(
+    'FormField composed with Select',
+    withInfo(
+      'This example demonstrates how the input field will behave with a select dropdown'
+    )(() => (
+      <Flex width={900}>
+        <Box width={1 / 3} pl={3}>
+          <Text bold pt={4} pb={2}>
+            User Types a Value
+          </Text>
+          <FormField onChange={() => {}}>
+            <Label>Cabin Class</Label>
+            <Icon name="key" color="blue" size="18" />
+            <Select
+              id="cabinClass"
+              name="cabinClass"
+              defaultValue="Premium Economy"
+              prefixIcon="key"
+            >
+              <option>Economy</option>
+              <option>Premium Economy</option>
+              <option>Business</option>
+              <option>First Class</option>
+              <option>
+                With a super long label that doesn't get clobbered by the
```

Comment:
Very trivial. Perhaps as a "_super long label_" example, the content inside the `option` tag can go for the "_extreme lorem ipsum_" with usage of a `Truncate` from design system to show interoperable composition with design system components composing each other in a Storybook example?
https://pricelinelabs.github.io/design-system/Truncate

Reply:
I don't think Truncate will work as expected inside an `<option>` element, which should automatically truncate however the OS handles it

Reply:
Oh thanks for educating me about this nuance of `<option>` with `Truncate`. Will keep that in mind when using `<option>`.